cpp/src/Slice/SliceUtil.cpp

```diff
@@ -321,7 +321,7 @@ Slice::filterMcppWarnings(const string& message)
 }
 
 void
-Slice::printGeneratedHeader(IceInternal::Output& out, const string& path, const string& comment)
```

Comment:
This variable was named `commentStyle` in the header file:
```
printGeneratedHeader(IceInternal::Output& out, const std::string& path, const std::string& commentStyle = "//");
```
I fixed this to use the same name. Header and source files should use the same name IMO.